### PR TITLE
Remove dispatcher from initialize method

### PR DESCRIPTION
**Pull request by daannijkamp**

### CODE REVIEW

**Comment by yahoocla** [comment]
Thank you for submitting this pull request, however I do not see a valid CLA on file for you. Before we can merge this request please visit https://yahoocla.herokuapp.com/ and agree to the terms. Thanks! :smile:

**Comment by yahoocla** [comment]
CLA is valid!

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 node_modules
 build
-npm-debug.log
\ No newline at end of file
+npm-debug.log
+*.js~
```
Comment (mridgway) [inline .gitignore]: This seems specific to your development environment and should be put in your ~/.gitignore instead of in the project's.

**Comment by mridgway** [comment]
Nice. Thanks!

**Pull request merged**